docker-compose.yml

```diff
@@ -22,6 +22,31 @@ services:
       - "8080:8080"
       - "8081:8081"
 
+  db:
+    image: postgres:latest
```

Comment:
Please change image tag `latest` to `16.0` so that we always know which version is actually used. Dependabot should create PRs for upgrading these images.